src/Stratis.Bitcoin.Features.Miner/PosMinting.cs

```diff
@@ -20,6 +20,39 @@
 
 namespace Stratis.Bitcoin.Features.Miner
 {
+    /// <summary>
+    /// PosMinting is used in order to generate new blocks. It involves a sort of lottery, similar to proof-of-work,
+    /// but the chances of winning this lottery depend on how many coins you are staking, not on hashing power.
```

Comment:
but the chances of winning this lottery is `proportional` to how many coins you are staking